packages/p2p/src/components/orders/orders.scss

```diff
@@ -248,4 +248,20 @@
             flex: 1;
         }
     }
+    &__banner {
+        margin-top: 1.6rem;
+        width: fit-content;
+
+        @include mobile {
+            margin: 1.6rem 1.6rem 0;
+
+            .dc-hint-box {
+                align-items: normal;
+
+                &__icon {
+                    width: 3.5rem;
+                }
+            }
```

Comment:
The component HintBox has a className prop, can you use that instead?

Reply:
Made the change